<ui version="4.0" >
 <class>QuickOpen</class>
 <widget class="QDialog" name="QuickOpen" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Quick Open</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QLabel" name="searchLabel" >
     <property name="text" >
      <string>Search:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="searchLine" />
   </item>
   <item>
    <widget class="QLabel" name="listLabel" >
     <property name="text" >
      <string>List:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ExpandingTree" name="list" >
     <property name="focusPolicy" >
      <enum>Qt::NoFocus</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="showAdvanced" >
     <property name="text" >
      <string>Show Advanced</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
     <property name="flat" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="options" >
     <property name="title" >
      <string>GroupBox</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <item>
       <widget class="QGroupBox" name="scopeGroup" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="title" >
         <string>Scope:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="flat" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="itemsGroup" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="title" >
         <string>Items:</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="flat" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>itemsGroup</zorder>
     <zorder>scopeGroup</zorder>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ExpandingTree</class>
   <extends>QTreeView</extends>
   <header>expandingtree/expandingtree.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>searchLine</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QuickOpen</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>252</x>
     <y>468</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QuickOpen</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>320</x>
     <y>468</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showAdvanced</sender>
   <signal>toggled(bool)</signal>
   <receiver>options</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>154</x>
     <y>358</y>
    </hint>
    <hint type="destinationlabel" >
     <x>152</x>
     <y>379</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
